### ISSUE TITLE
IKEA Floalt and Remote

### ISSUE THREAD

**Issue by Jogobo** [Koenkk/zigbee-herdsman-converters#1133]
I have a TRADFRI remote control bound to a group with a FLOALT panel WS 60x60 as group member (remote has old firmware and I was not able to update yet). When the remote control is paired directly with the panel the arrow buttons ('>' and '<') change the white temperature of the panel.
When panel and remote are paired with zigbee2mqtt the On/Off button (center) and the brightness buttons (small and large sun) work as expected. The arrow buttons seem to have no function. Looks like the buttons commands are not recognized by the panel.

**Comment by Koenkk** [comment]
This is indeed a known issue, when paired to the TRADFRI hub does some magic setup. Sniffing the Zigbee traffic while pairing a tradfri remote and a bulb with the Tradfri hub could give some insight on how to solve it. https://www.zigbee2mqtt.io/how_tos/how_to_sniff_zigbee_traffic.html

**Comment by Jogobo** [comment]
Can't help further on this issue. I don't have a TRADFRI hub as the FLOALT panel came together with the remote and was standalone until now. 
As I am bringing everything together under openHAB (Osram Lightify, Homematic, IKEA, Alexa) bit by bit I did so with my only two IKEA pieces yesterday, when I came across this issue.
Maybe I should unbind FLOALT and remote and configure them as separate things in openHAB. So openHAB can do the right stuff at the right time...
